### ISSUE TITLE
flutter2.0   Null safety

### ISSUE THREAD

**Issue by panzikeji** [flutter-moum/flutter_screenshot_callback#42]

**Comment by Joseph-Nathan** [comment]
any news?

**Comment by ousvat** [comment]
You can also use this package from my repo where I did more fixes:
```
screenshot_callback:
    git:
      url: git://github.com/ousvat/flutter_screenshot_callback.git
 ```

**Comment by beygee** [comment]
Fixed 2.0.0.
Sorry for being late.
